src/AppBundle/Repository/PostRepository.php

```diff
@@ -38,4 +38,24 @@ public function findLatest($limit = Post::NUM_ITEMS)
             ->getResult()
         ;
     }
+
+    public function findByTerms(array $terms, $limit = Post::NUM_ITEMS)
+    {
+        $queryBuilder = $this->createQueryBuilder('p');
```

Comment:
Symfony Best Practices don't mention about short variable's name, so I prefer to based on Symfony docs. If it'll be changed - so we could change it in Symfony demo